C/manual/tips-appendix.docbook

```diff
@@ -45,6416 +63,6935 @@
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>Finance::Quote Name
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>URL, Notes
+                  </para>
+                </entry>
+              </row>
+            </thead>
+
+            <tbody valign='middle'>
+              <row>
+                <entry>
+                  <para>American International Assurance, HK
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>aiahk
+                  </para>
+                </entry>
+
+                <entry>
+                  <para><ulink url="https://www.aia.com.hk"/>
+                  </para>
+                </entry>
+              </row>
+
+              <row>
+                <entry>
+                  <para>BAMOSZ funds, HU
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>bamosz
+                  </para>
+                </entry>
+
+                <entry>
+                  <para><ulink url="https://www.bamosz.hu"/>
+                  </para>
+                </entry>
+              </row>
+
+              <row>
+                <entry>
+                  <para>BMO NesbittBurns, CA
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>bmonesbittburns
+                  </para>
+                </entry>
+
+                <entry>
+                  <para><ulink url="https://bmonesbittburns.com"/>
+                  </para>
+                </entry>
+              </row>
+
+              <row>
+                <entry>
+                  <para>Budapest Stock Exchange (BET), ex-BUX, HU
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>bse or bet
+                  </para>
+                </entry>
+
+                <entry>
+                  <para><ulink url="https://www.bet.hu"/>
+                  </para>
+                </entry>
+              </row>
+
+              <row>
+                <entry>
+                  <para>Citywire Funds, GB
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>citywire
+                  </para>
+                </entry>
+
+                <entry>
+                  <para><ulink url="https://citywire.co.uk"/>
+                  </para>
+                </entry>
+              </row>
+
+              <row>
+                <entry>
+                  <para>Cominvest Asset Management, ex-Adig, DE
+                  </para>
+                </entry>
+
+                <entry>
+                  <para>cominvest
+                  </para>
+                </entry>
+
+                <entry>
+                  <para><ulink url="https://www.cominvest-am.de"/>
+                  </para>
+
+                  <para>Obsolete, use comdirect"/>
```

Comment:
The Commerzbank AG changed their structure, use comdirect now.

Reply:
done